lib/app_web/live/app_live.html.heex

```diff
@@ -1,4 +1,4 @@
-<div class="h-90 w-full font-sans">
+<main class="h-90 w-full font-sans">
```

Comment:
We will have have two `main` tags on the document, which I don't think is a valid html markdown:
![image](https://user-images.githubusercontent.com/6057298/184601335-5b037a15-a63d-4d74-a75f-f2d267ae6300.png)

The other `main` tag is defined in the `live` layout: https://github.com/dwyl/mvp/blob/main/lib/app_web/templates/layout/live.html.heex

Reply:
Good catch, just checked the source: 

![image](https://user-images.githubusercontent.com/194400/184602968-cfd3a5a3-4147-41f1-ae58-258945c5fa06.png)

I have my editor open will remove one of them now.

Reply:
Removed it. Thanks.